reference/forms/types/options/choice_translation_domain.rst.inc

```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7
```

Comment:
According to the original PR, this option was added in Symfony 2.8: https://github.com/symfony/symfony/commit/3909460